static/js/narrow.js

```diff
@@ -129,10 +129,12 @@ function update_narrow_title(filter) {
     // If the operator is something other than "stream", "topic", or
     // "is", we shouldn't update the narrow title
     if (filter.has_operator("stream")) {
+        const stream_name = filter.operands("stream")[0];
+        const topic_name = filter.operands("topic")[0];
```

Comment:
Let's define this inside the case where there is a `topic` operator. I'd kinda expect this line to crash in the stream-only narrow case, but maybe it just returns `undefined`?

Reply:
Makes sense. Moved topic_name under the filter topic operator, thanks!